### DISCUSSION TITLE
Is there a detailed introduction to the coupling mechanism and parallel computing algorithms in MOOSE, as well as the theory of solving large sparse matrices

### DISCUSSION THREAD

**Discussion by Ating24** [idaholab/moose#27970] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

<Your question here>

**Comment by GiudGiud** [comment]
Hello

PETSc manual / intros will be your best reference for solving problems with sparse matrices in moose
Coupling you can look at this tutorial for multiapps (e.g. not single-matrix approach)
https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial02_multiapps/presentation/index.html#/
And this example for regular coupling
https://mooseframework.inl.gov/getting_started/examples_and_tutorials/examples/ex03_coupling.html

Guillaume
